src/FakeItEasy/FakeItEasy.csproj

```diff
@@ -32,10 +32,20 @@
   <!-- Release -->
 
   <PropertyGroup Condition="'$(Configuration)' == 'Release'">
-    <DocumentationFile>bin\$(Configuration)\$(TargetFramework)\FakeItEasy.xml</DocumentationFile>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <DebugType>pdbonly</DebugType>
   </PropertyGroup>
+
+  <!-- .NET Standard 2.1 -->
+
+  <PropertyGroup Condition="'$(TargetFramework)' == 'netstandard2.1'">
+    <DefineConstants>$(DefineConstants);FEATURE_REFLECTION_GETASSEMBLIES;FEATURE_EXCEPTION_DISPATCH_INFO;FEATURE_ARRAY_EMPTY;FEATURE_PARAMETERINFO_CUSTOMATTRIBUTES_PROPERTY</DefineConstants>
+  </PropertyGroup>
```

Comment:
I thought about this too, but I had a different approach in mind: centralize the list of features per framework somewhere (e.g. Directory.Build.props) so that we don't have to repeat them in every project.
But negative constants would work too, with the advantage you described.
And both approaches could be combined.